diff --git a/Cargo.toml b/Cargo.toml
index 9bd2033..18f2610 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,7 +24,7 @@ memflow = { version = "0.1", git = "https://github.com/memflow/memflow", branch
 #memflow = { version = "0.1", path = "../memflow/memflow", default-features = false }
 log = { version = "0.4", default-features = false }
 pelite = { version = "0.9", default-features = false }
-goblin = { version = "0.4", default-features = false, features = ["pe32", "pe64"] }
+goblin = { version = "0.4", features = ["pe32", "pe64"] }
 widestring = { version = "0.5", default-features = false, features = ["alloc"] }
 no-std-compat = { version = "0.4", features = ["alloc"] }
 serde = { version = "1.0", default-features = false, optional = true, features = ["derive"] }
